dev/tools/notify-upstream-pins.sh

```diff
@@ -0,0 +1,113 @@
+
+#!/usr/bin/env bash
+
+# Script to notify upstreams that we need a tag to put in a platform/installer
+
+VERSION="8.13"
+MINOR_VERSION="-beta1"
+#MINOR_VERSION=".0"
+DATE="December 7, 2020"
+CC="\nCC: https://github.com/coq/coq/issues/12334"
+#CC="\n@coqbot column:...."
+REASON="bundled in the Windows installer"
+#REASON="bundled in the Coq platform"
+
+git show master:dev/ci/ci-basic-overlay.sh > /tmp/master-ci-basic-overlay.sh
+git show v${VERSION}:dev/ci/ci-basic-overlay.sh > /tmp/branch-ci-basic-overlay.sh
+
+# caveats:
+# - dev/ci/gitlab.bat has \r (windows)
+# - aactactics, gappa, HB, extlib have different names in ci
+# - menhir is not pinned but figures as an addon
+WINDOWS_ADDONS=$(grep addon= dev/ci/gitlab.bat \
```

Comment:
@SkySkimmer would recommend that you use lowercase for variables that are supposed to be local to this script. I don't know about the parameters above.